blueflood-http/src/test/java/com/rackspacecloud/blueflood/utils/DateTimeParserTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2013 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.utils;
+
+import com.rackspacecloud.blueflood.utils.DateTimeParser;
+import junit.framework.Assert;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+public class DateTimeParserTest {
+    @Test
+    public void testFromUnixTimestamp() {
+        long unixTimestamp = nowDateTime().getMillis() / 1000;
+
+        Assert.assertEquals(DateTimeParser.parse(Long.toString(unixTimestamp)),
+                nowDateTime());
+    }
+
+    @Test
+    public void testPlainTimeDateFormat() {
+        // %H:%M%Y%m%d
```

Comment:
Am I missing some co-relation between this date format in the comment and what formats we are testing in the tests or there isn't really any relation? If there is no relation, then we should remove this comment as it is confusing.